docs: improve troubleshooting commands

euank opened the pull request:
per title, just trying to improve the accuracy/usefulness of the suggested commands.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.kubernetes.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.kubernetes.io/reviews/kubernetes/heapster/1389)
<!-- Reviewable:end -->

euank committed:
docs: improve troubleshooting commands

Crassirostris commented:
@euank Web Admin Interface is deprecated and will be removed, grafana should be a default way to interact with influxdb ([proof](https://docs.influxdata.com/influxdb/v1.1/tools/web_admin/)). Maybe delete this section entirely?

Otherwise, it looks great, thank you very much!

euank committed:
influxdb: remove references to the deprecated UI

Per https://docs.influxdata.com/influxdb/v1.1/tools/web_admin/ the web
UI is deprecated and off by default.

This commit removes it from the documentation, though the Dockerfile and
configuration related to that still enable it for the moment.

euank commented:
Thanks for the heads up, I removed the line about the influxdb UI as well as the commented out service port for it.

euank commented on docs/influxdb.md:
````diff
@@ -26,13 +26,21 @@ Take a look at the [storage schema](storage-schema.md) to understand how metrics
+    ```
 
-3. If you find InfluxDB to be using up a lot of CPU or memory, consider placing resource restrictions on the `InfluxDB & Grafana` pod. You can add `cpu: <millicores>` and `memory: <bytes>` in the [Controller Spec](../deploy/kube-config/influxdb/influxdb-grafana-controller.yaml) and relaunch the controllers:
+1. If you find InfluxDB to be using up a lot of CPU or memory, consider placing resource restrictions on the `InfluxDB & Grafana` pod. You can add `cpu: <millicores>` and `memory: <bytes>` in the [Controller Spec](../deploy/kube-config/influxdb/influxdb-grafana-controller.yaml) and relaunch the controllers by running `kubectl apply -f deploy/kube-config/influxdb/influxdb-grafana-controller.yaml`  ...
````
In markdown, it still renders as a correctly numbered list. It's somewhat common to just use `1` for every item so that adding new internal items doesn't require renumbering all the lines after.

I realized I forgot to do that the first time when I had to renumber to address the previous nit.

I can change this if you insist, but I think it's slightly better markdown style to just have em all as `1.`

Crassirostris replied:
OK, sure, sorry!

Crassirostris commented:
LGTM

Crassirostris commented:
@piosz Thanks!

Crassirostris commented:
@euank Thanks!
